app/vmstorage/main.go

```diff
@@ -83,6 +83,10 @@ var (
 		"See https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/#track-ingested-metrics-usage")
 	cacheSizeMetricNamesStats = flagutil.NewBytes("storage.cacheSizeMetricNamesStats", 0, "Overrides max size for storage/metricNamesStatsTracker cache. "+
 		"See https://docs.victoriametrics.com/victoriametrics/single-server-victoriametrics/#cache-tuning")
+
+	idbPrefillStart = flag.Duration("storage.idbPrefillStart", time.Hour, "Whether to start prefill next indexDB before it's rotation. "+
```

Comment:
Flag description is odd.
I think it should better explain what this flag does and also recommend users when to or not to touch it.

Reply:
Updated